src/serviceWorker.js

```diff
@@ -0,0 +1,141 @@
+// This optional code is used to register a service worker.
```

Comment:
Since you deleted the serviceWorker import from index.js, you can also remove this file and any other files you are not using